app/src/main/java/net/squanchy/tweets/TweetsPageView.java

```diff
@@ -41,28 +41,17 @@ public TweetsPageView(Context context, AttributeSet attrs) {
     }
 
     public TweetsPageView(Context context, AttributeSet attrs, int defStyleAttr) {
-        this(context, attrs, defStyleAttr, 0);
+        super(context, attrs, defStyleAttr);
     }
 
-    public TweetsPageView(Context context, AttributeSet attrs, int defStyleAttr, int defStyleRes) {
-        super(context, attrs, defStyleAttr, defStyleRes);
-
-        super.setOrientation(VERTICAL);
+    @Override
+    protected void onFinishInflate() {
+        super.onFinishInflate();
 
-        Activity activity = unwrapToActivityContext(context);
+        Activity activity = unwrapToActivityContext(getContext());
```

Comment:
While I'm at it, I will also fix the other 2 page view, schedule and favorites that had the same flow